package.json

```diff
@@ -5,6 +5,7 @@
   "keywords": [
```

Comment:
Since this is a backwards incompatible change we need to bump the major version and probably start doing proper releases and release notes.